src/Styles.re

```diff
@@ -367,4 +374,4 @@ let mergeStyles: list(string) => string =
   styles => {
     /* styles->Belt.List.reduce("", mapStylesFunction);*/
     styles->Belt.List.reduce("", (prev, next) => prev ++ " " ++ next);
-  };
+  };
```

Comment:
@DenhamPreen We should get our editor settings more aligned. Mine adds a trailing newline on all files. I personally prefer this since it means that it is easier to add new things to this file when editing.